firmware/shared/src/hw/hw_chimera_v2.c

```diff
@@ -124,157 +150,208 @@ static const SpiDevice *hw_chimera_v2_getSpi(hw_chimera_v2_Config *config, const
 }
 #endif
 
-// Handle an rpc message.
-void hw_chimera_v2_handleContent(hw_chimera_v2_Config *config, uint8_t *content, uint16_t length)
+/**
+ * @brief Evaluate a Chimera V2 request.
+ * @param config Collection of protobuf enum to peripheral tables and net name tags.
+ * @param request Request from the Chimera V2 python client.
+ * @param response Pointer to the response struct to write the result to.
+ * @return True if success, otherwise false.
+ */
+bool hw_chimera_v2_evaluateRequest(hw_chimera_v2_Config *config, ChimeraV2Request *request, ChimeraV2Response *response)
```

Comment:
place static out front

and you don't need the long name, long name makes it seem like it's public